spec/controllers/editors_controller_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe EditorsController do
+
+  before { sign_in(user) }
+
+  let(:user) { FactoryGirl.create(:user) }
+  let(:paper) do
+    paper = FactoryGirl.create(:paper)
+    paper.paper_roles.create!(role: PaperRole::EDITOR, user: user)
+    paper
+  end
+
+  describe "DELETE /papers/:id/editor" do
+    it "removes the current paper editor" do
+      delete(:destroy, {
+        format: "json",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.